### PR TITLE
Set default indices and change sku index type to not_analyzed

### PR DESCRIPTION
**Pull request by arudnicka**
Resolve a problem with SSR on product page because of not existing sku

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -43,6 +43,43 @@ const getElasticClient = () => (
   })
 )
 
+const createIndeces = () => {
```
Comment (tniezg) [inline src/index.ts]:
```suggestion
const createIndices = () => {
```

File: src/index.ts [hunk 2]
```diff
@@ -77,10 +114,15 @@ const getSpreeClient = () => (
 const preconfigMapPages = (makePaginationRequest, resourceCallback) =>
   mapPages(makePaginationRequest, resourceCallback, paginationOptions.perPage, paginationOptions.maxPages)
 
+program.command('create-indeces')
+  .action(() => {
+    createIndeces()
```
Comment (tniezg) [inline src/index.ts]:
```suggestion
    createIndices()
```

File: src/index.ts [hunk 3]
```diff
@@ -77,10 +114,15 @@ const getSpreeClient = () => (
 const preconfigMapPages = (makePaginationRequest, resourceCallback) =>
   mapPages(makePaginationRequest, resourceCallback, paginationOptions.perPage, paginationOptions.maxPages)
 
+program.command('create-indeces')
```
Comment (tniezg) [inline src/index.ts]:
```suggestion
program.command('create-indices')
```

File: src/index.ts (hunk 1 above)
Comment (arudnicka) [reply src/index.ts]: 😅

**Review by tniezg — APPROVED** [review]

**Pull request merged**